lib/services/schedule.js

```diff
@@ -161,9 +165,10 @@ function post(uri, data, user) {
       return db.db.patchMeta(pageUri, meta);
     })
     .then(() => {
-      return bus.publish('schedulePage', { uri: pageUri, data, user });
+      bus.publish('schedulePage', { uri: pageUri, data, user });
+
+      return data;
     })
-    .then(() => data)
```

Comment:
are we sure this isn't a destructive change? we don't need to return the data returned by `del` on L106?

Reply:
That is still the case! See line 170. It's just expressed differently so that we don't do a return on the `bus.publish` (which returns undefined!)

Reply:
oops yeah right! looks good!